### ISSUE TITLE
CleanupDefunctSiloEntriesKey missing in Oracle Migrations

### ISSUE THREAD

**Issue by mkstephenson** [dotnet/orleans#8937]
I am attempting to start clustering on Orleans and get the error that the "CleanupDefunctSiloEntriesKey" entry is missing. Looking at the original PR that added the entries (#7928), the key for inserting the entry in the Oracle table is different ("DeleteMembershipTableEntriesKey" instead of "CleanupDefunctSiloEntriesKey"). Can I assume this is a copy-paste error and can be changed? Or is there a reason this key overrides an existing entry from the main clustering migration?

**Comment by m3nax** [comment]
Same problem for mssql #8676

**Comment by mkstephenson** [comment]
Yup, I saw that issue and it helped me with the MSSQL issue, but for Oracle the migration appears to have a typo.

**Comment by heku** [comment]
Following the [official guide](https://learn.microsoft.com/en-us/dotnet/orleans/host/configuration-guide/adonet-configuration), I still encounter the same problem. This issue has persisted for more than two years, which makes me question whether the repository is properly maintained.

**Comment by ReubenBond** [comment]
Resolved on main by https://github.com/dotnet/orleans/pull/11246. The Oracle 3.7.0 migration now registers CleanupDefunctSiloEntriesKey under the correct name and can be reapplied safely. The fresh-install clustering script also includes the query, so both installation paths provide the query required at startup. Closing as completed.
